languagetool-language-modules/de/src/main/java/org/languagetool/language/German.java

```diff
@@ -542,6 +543,43 @@ protected int getPriorityForId(String id) {
         // less prio than spell checker
         return -4;
       }
+      if (id.startsWith("AI_DE_GGEC_UNNECESSARY_PUNCTUATION")) {
+        // less prio than FALSCHES_ANFUEHRUNGSZEICHEN
+        return -2;
+      }
+
+      // gGEC IDs that should have less prio than rules with default prio
+      // e. g. ABKUERZUNG_FEHLENDE_PUNKTE
+      String[] ggecIds = {
+        "AI_DE_GGEC_REPLACEMENT_ADJECTIVE",
+        "AI_DE_GGEC_REPLACEMENT_ADVERB",
+        "AI_DE_GGEC_REPLACEMENT_NOUN",
+        "AI_DE_GGEC_REPLACEMENT_ORTHOGRAPHY_LOWERCASE",
+        "AI_DE_GGEC_REPLACEMENT_ORTHOGRAPHY_SPELL",
+        "AI_DE_GGEC_REPLACEMENT_OTHER",
+        "AI_DE_GGEC_REPLACEMENT_VERB",
+        "AI_DE_GGEC_REPLACEMENT_VERB_FORM",
+        "AI_DE_GGEC_UNNECESSARY_ORTHOGRAPHY_SPACE",
+        "AI_DE_GGEC_UNNECESSARY_OTHER",
+        "AI_DE_GGEC_UNNECESSARY_SPACE"
+      };
+
+      for (String gId: ggecIds) {
+        if (id == gId) {
+          return -1;
+        }
+      }
+
+      Pattern pattern = Pattern.compile("AI_DE_GGEC_MISSING_PUNCTUATION_\\d+_DASH_J_HRIG|" +
```

Comment:
`DASH_J_HRIG` --> fix for incorrectly displayed Umlaute is in the progress, would it be possible to implement it as `DASH_J(_|AE)HRIG`?